### ISSUE TITLE
Tons of threads running even if -r 3

### ISSUE THREAD

**Issue by lukeomatik** [madMAx43v3r/chia-plotter#143]
Is it normal? In htop I see it spawned 16 threads and I have a load average of 8 (1m/5m/15m). My little i3 (2c/4t) is asking me pity.

Compiled in a fresh debian and copied the binary in my main archlinux machine.

**Comment by madMAx43v3r** [comment]
yeah the `-r` parameter is just a multiplier, phase 1 will launch about 3-4 times as many threads, but they wont all be fully loaded.

**Comment by zivester** [comment]
phase 1 on my machine with `-r 4` spikes the main process to 1000% (10x), will see how it calms down in later phases.

**Comment by digitalscream** [comment]
On Linux, you can use taskset to prevent it escaping the CPU boundaries you set. For example...

taskset -ac 0-2 chia_plot <keys> <paths> -r 3 -n 1

...will only allow it to use the physical cores (CPUs 0 and 1) and one HT core (CPU 2).
